interfaces.go

```diff
@@ -281,4 +281,5 @@ type Span struct {
 	Tags           map[string]string `json:"tags,omitempty"`
 	StartTimestamp time.Time         `json:"start_timestamp"`
 	EndTimestamp   time.Time         `json:"timestamp"`
+	Data           map[string]string `json:"data,omitempty"`
```

Comment:
The values are not limited to strings.

```suggestion
	Data           map[string]interface{} `json:"data,omitempty"`
```

(gofmt will likely change the surrounding lines too for alignment, so do not apply the suggestion on GH)

Reply:
Good catch, fixed.